pkg/controller/admissionchecks/multikueue/multikueuecluster.go

```diff
@@ -98,6 +98,7 @@ func newRemoteClient(localClient client.Client, wlUpdateCh, watchEndedCh chan<-
 		localClient:  localClient,
 		origin:       origin,
 	}
+	rc.connecting.Store(true)
```

Comment:
Is this needed for the skip itself or it fixes another scenario?

Reply:
It is needed, it's one of the two key parts of the fix.

Reply:
ack